airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/OrchestratorConstants.java

```diff
@@ -29,7 +30,26 @@
       EnvConfigs.JOB_MAIN_CONTAINER_MEMORY_REQUEST,
       EnvConfigs.JOB_MAIN_CONTAINER_MEMORY_LIMIT,
       EnvConfigs.JOB_DEFAULT_ENV_MAP,
-      EnvConfigs.LOCAL_ROOT);
+      EnvConfigs.LOCAL_ROOT,
+      "LOG_LEVEL", // todo: put in constant
+      LogClientSingleton.GCS_LOG_BUCKET,
+      LogClientSingleton.GOOGLE_APPLICATION_CREDENTIALS,
+      LogClientSingleton.S3_MINIO_ENDPOINT,
+      "S3_PATH_STYLE_ACCESS", // todo: put in constant
```

Comment:
assuming these will be done before merge?

Reply:
I removed these from the env config map since we previously were not consuming them